packages/api/src/auth/github/github.controller.spec.ts

```diff
@@ -4,13 +4,14 @@ import httpMocks from 'node-mocks-http';
 
 import { env } from '@/common/env';
 
+import { UserId } from '../../users/users.types';
 import { JwtService } from '../jwt/jwt.service';
 import { GithubController } from './github.controller';
 import { GithubService } from './github.service';
 import { GithubAuthGuardReq } from './github.types';
 
 const user = {
-  id: 1,
+  id: '1',
```

Comment:
Would be great to use in tests some real values, so here generate uuid. 
https://www.guidgenerator.com/